### ISSUE TITLE
Popup Currently Not Working on Android

### ISSUE THREAD

**Issue by rsimon** [recogito/text-annotator-js#185]
Hm... looks like the popup from the `@recogito/react-text-annotator` package is currently not working on Android. (Seems to work ok on desktop + iOS.)

**Comment by oleksandr-danylchenko** [comment]
Yeah, I can reproduce it locally 👀

**Comment by oleksandr-danylchenko** [comment]
Yeah, on creation the `bounds` are undefined for the annotation 🤔 
![image](https://github.com/user-attachments/assets/a801107a-cff6-42c4-9538-498999d8cf95)
https://github.com/recogito/text-annotator-js/blob/5b9abad31724db14d473f7d363b295a967cd93b7/packages/text-annotator-react/src/TextAnnotationPopup/TextAnnotationPopup.tsx#L95-L98

**Comment by oleksandr-danylchenko** [comment]
I think it went unnoticed in my app, as I also added the "idling" check before showing the popup - https://github.com/recogito/text-annotator-js/pull/174. So it gives enough time for the `bounds` to get populated before we query them.
